recipes/tiny-aes-c/all/conanfile.py

```diff
@@ -0,0 +1,94 @@
+import os
+import glob
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.32.0"
+
+class TinyAesCConan(ConanFile):
+    name = "tiny-aes-c"
+    version = "1.0.0"
+    license = "The Unlicense"
```

Comment:
```suggestion
    license = "Unlicense"
```

https://spdx.org/licenses/Unlicense.html